taxi/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
The `tests.py` file is empty. Consider adding test cases for your models and any other functionality you want to ensure works correctly. This is crucial for maintaining code quality and reliability.